R/fortify-map.r

```diff
@@ -39,16 +39,16 @@ fortify.map <- function(model, data, ...) {
 
 #' Create a data frame of map data
 #'
-#' Easily turn data from the \pkg{maps} package in to a data frame suitable
+#' Easily turn data from the \pkg{maps} package into a data frame suitable
 #' for plotting with ggplot2.
 #'
-#' @param map name of map provided by the \pkg{maps} package.  These
+#' @param map name of map provided by the \pkg{maps} package. These
 #'   include [maps::county()], [maps::france()],
 #'   [maps::italy()], [maps::nz()],
 #'   [maps::state()], [maps::usa()],
 #'   [maps::world()], [maps::world2()].
-#' @param region name of subregions to include.  Defaults to `.` which
-#'   includes all subregion.  See documentation for [maps::map()]
+#' @param region name of subregion(s) to include. Defaults to `.` which
```

Comment:
For consistency, "name(s)"?

e.g. 
https://github.com/tidyverse/ggplot2/blob/acaa98e4cfcd922a737f35c16ec3dd6079aab3dc/R/scale-hue.r#L9

Reply:
Thanks @yutannihilation , that makes sense. I amended the commit to include your suggestion.